client/src/store/analyze-name-module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module, VuexModule, getModule, Mutation, Action } from 'vuex-module-decorators'
```

Comment:
We implemented VueX differently. I'm not sure which is better. Have a look: https://github.com/bcgov/lear/tree/master/coops-ui/src/store